cmd/archeio/main.go

```diff
@@ -17,58 +17,58 @@ limitations under the License.
 package main
 
 import (
-	"log"
+	"flag"
 	"net/http"
 	"os"
-	"regexp"
 	"strings"
+
+	"k8s.io/klog/v2"
 )
 
 func main() {
+	// cloud run expects us to listen to HTTP on $PORT
 	port := os.Getenv("PORT")
 	if port == "" {
 		port = "8080"
 	}
-	log.Printf("listening on port %s", port)
+
+	// klog setup
+	klog.InitFlags(nil)
+	flag.Parse()
+	defer klog.Flush()
+
+	// actually serve traffic
+	klog.V(0).InfoS("listening", "port", port)
```

Comment:
Nit: any reason to use V(0) - it's the default, right?

Reply:
I wasn't sure at first, we should probably drop the `V(0)` but it only runs once at least.